packages/docs/components/toast.md

```diff
@@ -0,0 +1,374 @@
+::: slot nimatron-all
+
+<ToastPen ref="toastBox"/>
+
+# Toast
+
+Toasts are non-disruptive messaging UIs that appear at the bottom right of the interface to provide quick, at-a-glance feedback on the outcome of an action.
+
+<figure class="nimatron--example">
+  <div class="nimatron--rendered">
+    <aside role="status" class="ods-toast is-sample-animation-off">
+      <span class="ods-toast--icon">
+        <svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon" aria-hidden="true"><path fill-rule="evenodd" clip-rule="evenodd" d="M7 13C10.3137 13 13 10.3137 13 7C13 3.68629 10.3137 1 7 1C3.68629 1 1 3.68629 1 7C1 10.3137 3.68629 13 7 13ZM8 4C8 4.55228 7.55228 5 7 5C6.44772 5 6 4.55228 6 4C6 3.44772 6.44772 3 7 3C7.55228 3 8 3.44772 8 4ZM8 6V11H6V6H8Z" fill="currentColor"/></svg>
+      </span>
+      <h1 class="ods-toast--title">
+        This is an info toast
+      </h1>
+      <p class="ods-toast--body">This is the default variant. It provides the user with neutral information.</p>
+      <button class="ods-toast--dismiss" aria-label="Dismiss toast" data-micromodal-close></button>
+    </aside>
+  </div>
+
+```html
+<aside role="status" class="ods-toast is-sample-animation-off">
+  <span class="ods-toast--icon">
+    <svg viewBox="0 0 14 14" fill="none" xmlns="http://www.w3.org/2000/svg" class="ods-icon" aria-hidden="true">...</svg>
+  </span>
+  <h1 class="ods-toast--title">
+    This is an info toast
+  </h1>
+  <p class="ods-toast--body">
+    This is the default variant. It provides the user with neutral information.
+  </p>
+  <button
+    class="ods-toast--dismiss"
+    aria-label="Dismiss toast"
+  ></button>
+</aside>
+```
+
+</figure>
+
+## Behavior
+
+Toasts may be triggered by many different kinds of events, but they are always transient (self-dismissing). With this in mind, Toasts should not include any interactive or long-form content.
+
+The Toast pen, `.ods-toast--pen`, will take care of positioning and layout automatically. Toasts will appear in the bottom-right corner of the viewport in the topmost content layer. 
+
+If multiple Toasts are triggered at once, they will stack in order of appearance. In addition, when multiple Toasts are present, they will resize to match the largest Toast onscreen, ensuring visual consistency.
+
+## Anatomy
+
+Each Toast is made of up to four components:
+
+<dl>
+    <dt>Icon &amp; color</dt>
+    <dd>These are mapped to the different variants of Toast and shouldn't be mixed. See below for more details.</dd>
+    <dt>Dismiss "X"</dt>
+    <dd>Included to ensure users are able to dismiss Toasts manually. Placement is handled automatically based on content.</dd>
```

Comment:
Not sure if this is intended, but why is the `focus` border color always the same?

| Type | Image |
| ----- | ----- |
| Info | <img width="468" alt="Screen Shot 2020-09-30 at 11 28 32 AM" src="https://user-images.githubusercontent.com/20212241/94725271-5841cf00-0310-11eb-9380-925bd7ed7277.png"> |
| Success | <img width="484" alt="Screen Shot 2020-09-30 at 11 28 42 AM" src="https://user-images.githubusercontent.com/20212241/94725234-4bbd7680-0310-11eb-9ef8-63eb3258f5c3.png">|
| Caution | <img width="466" alt="Screen Shot 2020-09-30 at 11 28 52 AM" src="https://user-images.githubusercontent.com/20212241/94725216-4829ef80-0310-11eb-965d-fa732c7203c6.png"> |
| Danger | <img width="480" alt="Screen Shot 2020-09-30 at 11 29 00 AM" src="https://user-images.githubusercontent.com/20212241/94725209-44966880-0310-11eb-9399-58bd1dca2cc5.png"> |

Reply:
We have a followup ticket to create a standard "Dismiss X". 

We don't want the `:focus` color to be programmatic because we can't ensure it'll always meet a11y. We also don't want to be manually curating it for every component.